kingpin/test/test_utils.py

```diff
@@ -33,6 +33,20 @@ def test_populate_with_env(self):
         result = utils.populate_with_tokens(string, tokens)
         self.assertEquals(result, expect)
 
+    def test_populate_with_values_not_default(self):
+        tokens = {'UNIT_TEST': 'FOOBAR', 'SECOND_UNIT': 'BARBAR'}
+        string = 'Unit %UNIT_TEST?DEFAULT% Test %SECOND_UNIT?DEFAULT2%'
```

Comment:
We've been discussing that for a long time... I don't really care, and i'm even ok with allowing multiple values. could be `|` or double: `||` could even be "space or space" ` or ` `?=` `:` ... don't know